sysutils/puppet-agent/src/opnsense/mvc/app/models/OPNsense/PuppetAgent/PuppetAgent.xml

```diff
@@ -0,0 +1,21 @@
+<model>
+    <mount>//OPNsense/puppetagent</mount>
+    <description>
+        Manage Puppet Agent service
+    </description>
+    <items>
+        <!-- container -->
+        <general>
+            <!-- fields -->
+            <Enabled type="BooleanField">
+                <default>0</default>
+                <Required>Y</Required>
+            </Enabled>
+            <FQDN type="TextField">
```

Comment:
HostnameField please

Reply:
I've changed it from TextField to HostnameField.